### DISCUSSION TITLE
Claimed Ticket category

### DISCUSSION THREAD

**Discussion by BlueSky-fur** [Sayrix/Ticket-Bot#138] [Ideas]
For better oversight for staff-members i was thinking of the following idea

When you open a ticket it gets moved into the Open Tickets category as setup per user.

As soon as it gets claimed by a staff member the ticket could move into a new category (if choosen to setup in the config)
and the ticket then switches from its standard name to the name of the staffmember and the user that created the ticket

Example

Freshly Opened ticket name
`💌︱8・bluesky` 

when claimed would switch to a diffrent category if choosen so in config and then rename to

`💌︱staffusername・bluesky` 

Thanks in advance for your time to read this!

**Comment by BlueSky-fur** [comment]
Was this ever implemented @Sayrix ?

**Reply by zhiyan114** [reply]
I think he probably only implemented the name changing part, not the category switching part. But if you still need it, you can open up an issue and I'll get it added in the next release.

**Reply by zhiyan114** [reply]
Actually, I don't even know how he implemented this. But from the look, it def not how you wanted it.
Future inquiries about this can be moved to #168
